pkg/objectstate/rte/rte.go

```diff
@@ -91,19 +91,17 @@ func (em *ExistingManifests) MachineConfigsState(mf rtemanifests.Manifests) []ob
 				klog.Warningf("failed to find machine config %q under the namespace %q", mcName, desiredMachineConfig.Namespace)
 				continue
 			}
+			os := objectstate.ObjectState{
+				Existing: existingMachineConfig.machineConfig,
+				Error:    existingMachineConfig.machineConfigError,
+				Desired:  desiredMachineConfig,
+				Compare:  compare.Object,
+				Merge:    merge.ObjectForUpdate,
+			}
 
-			ret = append(ret,
-				objectstate.ObjectState{
-					Existing: existingMachineConfig.machineConfig,
-					Error:    existingMachineConfig.machineConfigError,
-					Desired:  desiredMachineConfig,
-					Compare:  compare.Object,
-					Merge:    merge.ObjectForUpdate,
-				},
-			)
+			ret = append(ret, os)
```

Comment:
why this change?

Reply:
residues of an old implementation effort. Removed the changes